docs/hacking/install.rst

```diff
@@ -131,15 +116,15 @@ system that Docker can be installed on:
 
    .. code-block:: bash
 
-      docker run -d --name nsqd -p 4150:4150 -p 4151:4151 nsqio/nsq /nsqd
-      docker run -d --name postgres -p 5432:5432 postgres
-      docker run -d --name redis -p 6379:6379 redis
-      docker run -d --name elasticsearch -p 9200:9200 -p 9300:9300 nickstenning/elasticsearch-icu
+      sudo docker run -d --name nsqd -p 4150:4150 -p 4151:4151 nsqio/nsq /nsqd
```

Comment:
I think adding a sentence that says "you may need to run `docker` commands as root" would be enough.